packages/sources/lotus/test/e2e/balance.test.ts

```diff
@@ -11,20 +11,25 @@ describe('execute', () => {
     const requests = [
       {
         name: 'id not supplied',
-        testData: { data: { addresses: ['f2eaaj6w4evrdscw4s4o5c3df7ph725tbs3yvg6gi'] } },
+        testData: {
+          data: { addresses: [{ address: 'f2eaaj6w4evrdscw4s4o5c3df7ph725tbs3yvg6gi' }] },
+        },
       },
       {
         name: 'single address supplied',
-        testData: { id: jobID, data: { addresses: ['f2eaaj6w4evrdscw4s4o5c3df7ph725tbs3yvg6gi'] } },
+        testData: {
+          id: jobID,
+          data: { addresses: [{ address: 'f2eaaj6w4evrdscw4s4o5c3df7ph725tbs3yvg6gi' }] },
+        },
       },
       {
         name: 'multiple addresses supplied',
         testData: {
           id: jobID,
           data: {
             addresses: [
-              'f2eaaj6w4evrdscw4s4o5c3df7ph725tbs3yvg6gi',
-              'f225ey7bq53ur6sgrkxgf74hl2ftxkajupatwnmay',
+              { address: 'f2eaaj6w4evrdscw4s4o5c3df7ph725tbs3yvg6gi' },
```

Comment:
Similar to how the change to the `chain-reserve-wallet` response affects the `ada-balance` adapter, will changing this response also affect the downstream adapter in the PoR flow?  I'm looking at `balance.ts` in the PoR adapter directory and it looks like we just pass in the response from this adapter to when constructing the request payload to the second adapter if we are not pulling wBTC balances.  Do we have to do a similar object mapping like we did to the `ada-balance` adapter?

```
{
          id: input.jobRunID,
          data: {
            result: input.data.result,
            dataPath: 'result',
            endpoint: 'balance',
            confirmations,
          },
        }
 ```